server/src/instant/model/schema.clj

```diff
@@ -29,98 +29,116 @@
-                         current-attr (get-in current-schema [:refs link-desc])
-                         new-attr? (not current-attr)
-                         unchanged-attr? (and
-                                          (= (get new-attr :cardinality) (get current-attr :cardinality))
-                                          (= (get new-attr :unique?) (get current-attr :unique?))
-                                          (= (get new-attr :on-delete) (get current-attr :on-delete))
-                                          (= (get new-attr :on-delete-reverse) (get current-attr :on-delete-reverse)))]
-                     (cond
-                       unchanged-attr? nil
-                       new-attr? [:add-attr
-                                  {:value-type :ref
-                                   :id (UUID/randomUUID)
-                                   :forward-identity [(UUID/randomUUID) from-ns from-attr]
-                                   :reverse-identity [(UUID/randomUUID) to-ns to-attr]
-                                   :cardinality (:cardinality new-attr)
-                                   :unique? (:unique? new-attr)
-                                   :index? (:index? new-attr)
-                                   :on-delete (:on-delete new-attr)
-                                   :on-delete-reverse (:on-delete-reverse new-attr)}]
-                       :else [:update-attr
-                              {:value-type :ref
-                               :id (:id current-attr)
-                               :forward-identity (:forward-identity current-attr)
-                               :reverse-identity (:reverse-identity current-attr)
-                               :cardinality (:cardinality new-attr)
-                               :unique? (:unique? new-attr)
-                               :index? (:index? new-attr)
-                               :on-delete (:on-delete new-attr)
-                               :on-delete-reverse (:on-delete-reverse new-attr)}])))
-                 new-refs)
-        steps  (->> (concat eid-ops blob-ops ref-ops)
-                    (filter some?)
-                    vec)]
+        blob-ops (for [[ns-name attrs] new-blobs
+                       [attr-name new-attr] attrs
+                       :let [current-attr      (get-in current-schema [:blobs ns-name attr-name])
+                             changed-type?     (and check-types?
+                                                    (not= (get new-attr :checked-data-type)
+                                                          (get current-attr :checked-data-type)))
+                             changed-unique?   (not= (get new-attr :unique?) (get current-attr :unique?))
+                             changed-index?    (not= (get new-attr :index?) (get current-attr :index?))
+                             changed-required? (not= (get new-attr :required?) (get current-attr :required?))
```

Comment:
I was doing some dogfooding, but wasn't able to get this to trigger. 

https://github.com/instantdb/instant/compare/required-params-3...required-params-dogfood?expand=1

Steps: 
1. I added `required` to a column in `cli-nodejs/db/instant.schema.ts`
2. Ran npx instant-cli push (this uses the latest local version)
3. The backend does not seem to pick this up

Reply:
Dohp, @tonsky ignore this. 

For some reason I thought this change was already in the backend. Switching to local backend, everything worked great 👍 👍